pkg/csi/service/common/topology.go

```diff
@@ -163,11 +163,11 @@ func GetHostsForSegment(ctx context.Context, topoSegment map[string]string, vCen
 				}
 				for _, childType := range folder.ChildType {
 					switch childType {
-					case "vim.Datacenter":
+					case "Datacenter":
```

Comment:
Why do we need this change ? Wasn't vim.Datacenter working before ?

Reply:
It was not tested in 2 level topology pipeline. This was a recent code change and it is going through BR pipelines for the first time.